lib/pluginmanager/install_strategy/elastic_xpack.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: utf-8
+# In the context of the plugin manager no dependencies are currently loaded.
+require_relative "../../../logstash-core/lib/logstash/version"
+require "pluginmanager/xpack/remote_installer"
+require "pluginmanager/utils/http_client"
+require "pluginmanager/ui"
+require "net/http"
+require "uri"
+
+module LogStash module PluginManager module InstallStrategy
+  class ElasticXpack
```

Comment:
If I understand correctly you want me to rename theses classes more into PackFetchStrategy, RepositoryStrategy, UriStrategy so they reflect more what they actually do?